src/components/VideoPlayerPreview/types.ts

```diff
@@ -0,0 +1,26 @@
+import type {GestureResponderEvent} from 'react-native';
```

Comment:
You don't need to include types.ts file, instead move these types to VideoPlayerThumbnail and index.tsx respectively